ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Block.hs

```diff
@@ -16,90 +16,37 @@ module Ouroboros.Consensus.Cardano.Block (
     -- checks from GHC. But GHC expects a data type, not a type family, that's
     -- why we sometimes mention the data type of the instance in these exports
     -- instead of the abstract type family.
-  , HardForkBlock (
```

Comment:
@EncodePanda Is there an option for preserving the way this export-list was? It's not a dealbreaker, but I prefer the way it was.

I'm of two minds about alphabetizing these. For the most part, I would expect such lists to be alphabetized. But the era names are the one example where the non-alphabetical ordering is intentional. (Though I don't know if that's ultimately the wisest choice.)

Reply:
This is unfortunately not supported by stylish. It should be rather simple to add though. I've created a ticket for it  https://github.com/jaspervdj/stylish-haskell/issues/358

Reply:
We're biting this bullet. Not a big deal.